prerequisites.py

```diff
@@ -0,0 +1,411 @@
+    print("===== Done")
+
+    print("""
+   ###########################################################################
+   ###                                                                     ###
+   ###    Remember that you must now add yourself to the cmsuser group:    ###
+   ###                                                                     ###
+   ###       $ sudo usermod -a -G cmsuser <your user>                      ###
+   ###                                                                     ###
+   ###    You must also logout to make the change effective.               ###
+   ###                                                                     ###
+   ###########################################################################
+    """)
+
+
+def uninstall_all():
+    """This function deletes all that was installed by the install() function:
+    - deletion of the cmsuser user
+    - deletion of isolate
+    - deletion of localization files
+    - deletion of configuration files
+    and so on.
+
+    """
+
+    check_root(True)
+
+    print("===== Deleting isolate from /usr/local/bin/")
+    try_delete(os.path.join(USR_ROOT, "bin", "isolate"))
+
+    print("===== Deleting configuration to /usr/local/etc/")
+    if ask("Type Y if you really want to remove configuration files: "):
+        for conf_file_name in ["cms.conf", "cms.ranking.conf"]:
+            try_delete(os.path.join(USR_ROOT, "etc", conf_file_name))
+
+    print("===== Deleting localization files")
+    for locale in glob(os.path.join("cms", "server", "po", "*.po")):
+        country_code = re.search(r"/([^/]*)\.po", locale).groups()[0]
+        print("  %s" % country_code)
+        dest_path = os.path.join(USR_ROOT, "share", "locale",
+                                 country_code, "LC_MESSAGES")
+        try_delete(os.path.join(dest_path, "cms.mo"))
+
+    print("===== Deleting empty directories")
+    dirs = [os.path.join(VAR_ROOT, "log"),
+            os.path.join(VAR_ROOT, "cache"),
+            os.path.join(VAR_ROOT, "lib"),
+            os.path.join(VAR_ROOT, "run"),
+            os.path.join(USR_ROOT, "include"),
+            os.path.join(USR_ROOT, "share")]
+    for _dir in dirs:
+        if os.listdir(_dir) == []:
+            try_delete(_dir)
+
+    print("===== Deleting Polygon testlib")
+    try_delete(os.path.join(USR_ROOT, "include", "cms", "testlib.h"))
+
+    print("===== Deleting user and group cmsuser")
+    try:
+        for user in grp.getgrnam("cmsuser").gr_mem:
+            os.system("gpasswd -d %s cmsuser" % (user))
+        os.system("userdel cmsuser")
+    except KeyError:
+        print("[Warning] Group cmsuser not found")
+
+    print("===== Done")
+
+
+USAGE = """%s <command>
+
+Available commands:
+- compile_l10n
+- compile_isolate
+- install_l10n  (requires root)
+- install_isolate  (requires root)
+- install  (requires root)
+- uninstall  (requires root)
+""" % (sys.argv[0])
+
+
+if __name__ == "__main__":
```

Comment:
Please use argparse, e.g.: http://chase-seibert.github.io/blog/2014/03/21/python-multilevel-argparse.html (but with an explicit dictionary command_name -> function instead of the crazy "let's just execute whatever function the user asks for").